cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -1361,6 +1361,26 @@ gdv_int32 ascii_utf8(const char* data, gdv_int32 data_len) {
   return static_cast<gdv_int32>(data[0]);
 }
 
+// Returns the ASCII character having the binary equivalent to A.
+// If A is larger than 256 the result is equivalent to chr(A % 256).
+GANDIVA_EXPORT
+const char* chr_int32(gdv_int64 context, gdv_int32 in, gdv_int32* out_len) {
+  in = in % 256;
+  if (in <= 0) {
```

Comment:
What about 0? Why is this returning an empty string for 0? Hive UDF is not returning an empty string for 0

Reply:
It now returns the '\0' character which represents the NULL character